src/refactor/index.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+
+// return on each Click the value of Square
+function Square(props) {
+  return (
+    <button className="square" onClick={props.onClick}>
+      {props.value}
+    </button>
+  )
+}
+// Describe Board Game + set boolean to setup user 
+class Board extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      squares: Array(9).fill(null),
+      xIsNext: true,
+    };
+  }
```

Comment:
You dont need a constructor to initiate a state, u could simply do:
```
class Board extends React.Component {
    state = {
      squares: Array(9).fill(null),
      xIsNext: true,
    };
}
```

Reply:
nice! 
but there is a big difference?

Reply:
Big? no but:
1. Every piece of code requires maintains, the dev who will come after you won't understand the reasoning for you to do so.
2. when I'm looking at such a redundant code It seems like you as a dev missing knowledge, since we want you to stand up as a future candidate in an interview we better sharpen any dev skill or knowledge.